content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -2,66 +2,76 @@
+
+   .. image:: convert/convert-leads-leads-button.png
+       :align: center
+       :alt: Leads menu on CRM application.
 
 Convert a lead into an opportunity
 ==================================
 
-When you click on a *Lead* you will have the option to convert it to
-an opportunity and decide if it should still be assigned to the same
-channel/person and if you need to create a new customer.
+To convert a lead into an *opportunity*,
+When you click on a *Lead* you will have the option to convert it to an opportunity and decide if it
+should still be assigned to the same channel/person and if you need to create a new customer.
 
-.. image:: convert/convert03.png
+.. image:: convert/convert-leads-convert-opp-button.png
     :align: center
+    :alt: Create opportunity button.
 
-If you already have an opportunity with that customer Odoo will
-automatically offer you to merge with that opportunity. In the same
-manner, Odoo will automatically offer you to link to an existing
-customer if that customer already exists.
+.. image:: convert/convert-leads-conversion-action.png
+    :align: center
+    :alt: Create opportunity pop-up.
 
-Merge opportunities
-===================
+If you already have an opportunity with that customer Odoo will automatically offer you to merge
+with that opportunity. In the same manner, Odoo will automatically offer you to link to an existing
+customer if that customer already exists.
 
-Odoo will also automatically propose to merge opportunities if they have
-the same email address. When merging opportunities, Odoo merges the
-information into the opportunity which was created first, giving
-priority to the information present on the first opportunity.
+Merge leads and opportunities
+=============================
 
-No information is lost: data from the other opportunity is logged in the
-chatter and the information fields for easy access.
+Odoo will also automatically propose to merge opportunities if they have the same email address.
+When merging opportunities, Odoo merges the information into the opportunity which was created
+first, giving priority to the information present on the first opportunity.
 
-.. image:: convert/convert_04.png
+.. image:: convert/convert-leads-similar-leads.png
     :align: center
+    :alt: Similar leads smart button.
+
+No information is lost: data from the other opportunity is logged in the chatter and the information
+fields for easy access.
 
-Would you find a duplicate yourself, ...you can also merge opportunities
-or leads even if the system doesn't propose it.
+Would you find a duplicate yourself, ...you can also merge opportunities or leads even if the system
+doesn't propose it.
 
-Here’s how, from the list view. Select the opportunities or leads you
-want to merge and the action button will appear. Then, you can select merge.
+Here's how, from the list view. Select the opportunities or leads you want to merge and the action
+button will appear. Then, you can select merge.
```

Comment:
```suggestion
To manually merge opportunities, select the opportunities (or leads) to merge from the
:guilabel:`Leads` page, by checking the boxes next to the desired opportunities (or leads).

Then, click the :guilabel:`Action ⚙️ (gear)` icon at the top of the page, to reveal a drop-down
menu. From that drop-down menu, select the :guilabel:`Merge` option to merge the selected
opportunities (or leads).

When :guilabel:`Merge` is selected from the :guilabel:`Action ⚙️` drop-down menu, a
:guilabel:`Merge` pop-up window appears. In that pop-up window, decide to :guilabel:`Assign
opportunities to` a :guilabel:`Salesperson` and/or :guilabel:`Sales Team`.

Below those fields, the :guilabel:`Select Leads/Opportunities` to merge, along with their related
information, can be found.

To merge those selected leads/opportunities, click :guilabel:`Merge`.
```
just a bit more detailed of an explanation -- this could also benefit from a screenshot of the Merge pop-up window that appears.